packages/skills/ml_train/handlers.py

```diff
@@ -84,32 +85,44 @@ def _handle_terms(self, msg: MLTradeMessage, sender: str):
         amount = terms.values["price"]
         address = terms.values["address"]
 
-        tx_msg = TransactionMessage(performative=TransactionMessage.Performative.PROPOSE,
-                                    skill_id=self.context._skill.config.name,
-                                    transaction_id="transaction0",
-                                    sender=self.context.agent_public_keys[ledger_id],
-                                    counterparty=address,
-                                    is_sender_buyer=True,
-                                    currency_pbk=terms.values['currency_pbk'],
-                                    amount=amount,
-                                    sender_tx_fee=self.context.strategy.max_tx_fee,
-                                    counterparty_tx_fee=terms.values["seller_tx_fee"],
-                                    # TODO the following parameter should be removed and refactored properly.
-                                    quantities_by_good_pbk={},
-                                    ledger_id=ledger_id,
-                                    # this is used to retrieve the opponent address later
-                                    dialogue_label=DialogueLabel(('',''), sender, self.context.agent_public_key).json,
-                                    # this is used to send the terms later - because the seller is stateless and must know
-                                    # what terms have been accepted
-                                    terms=terms)
-        self.context.decision_maker_message_queue.put_nowait(tx_msg)
-        logger.info("[{}]: proposing the transaction to the decision maker. Waiting for confirmation ...".format(self.context.agent_name))
+        # tx_msg = TransactionMessage(performative=TransactionMessage.Performative.PROPOSE,
```

Comment:
Do you want me to implement the DM side of things? It would help me evolve the DM.